plaidml/op/op_test.cc

```diff
@@ -382,6 +382,38 @@ TEST_F(OpTest, ExplicitPaddingNoOp) {
   checkExact(program, {I_input}, {O_output});
 }
 
+TEST_F(OpTest, ExplicitPaddingEdge) {
+  auto I = Placeholder(DType::FLOAT32, {1, 2, 3}, "A");
+  auto O = op::explicit_padding(I, {1, 2, 1}, {1, 2, 1}).mode(op::PadMode::EDGE);
```

Comment:
Change one of these padding parameters to e.g. `{2, 1, 1}` to also test:

1.  That we aren't swapping lo and hi pads
2. That we aren't applying padding in reverse order (which a palindrome for both doesn't test)